<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/ana.ico</normaloff>../icon/ana.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>合约类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="contract_type">
       <item>
        <property name="text">
         <string>认购期权</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>认沽期权</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>开平方向</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="option_side">
       <item>
        <property name="text">
         <string>买入开仓</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>卖出开仓</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>平仓策略</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="close_strategy">
       <item>
        <property name="text">
         <string>交易信号平仓</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>到期日平仓</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>移仓频率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="change_feq">
       <item>
        <property name="text">
         <string>到期日</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1天</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2天</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3天</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>移仓条件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="move_condition">
       <item>
        <property name="text">
         <string>收盘价</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>开盘价</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>到期时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="interval">
       <item>
        <property name="text">
         <string>当月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>下月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>下季</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>隔季</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>行权价</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="strike_interval">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>低1个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>低2个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>低3个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>低4个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>低5个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>低6个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>低7个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>低8个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>低9个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>平值</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高1个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高2个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高3个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高4个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高5个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高6个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高7个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高8个价位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高9个价位</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>智能匹配</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="smart_match">
       <item>
        <property name="text">
         <string>否</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>是</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>数量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="volume">
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>保证金系数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="deposit_ratio"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Delta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="delta"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Gamma</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="gamma"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Theta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="theta"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Vega</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="vega"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Rho</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="rho"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_15">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>ivix</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="ivix"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_16">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
